test/verify/check-users

```diff
@@ -858,6 +858,75 @@ class TestAccounts(MachineCase):
         b.wait_not_present("#groups-create-dialog")
         b.wait_visible('#groups-list td[data-label="Group name"]:contains("saturn")')
 
+        # Deleting group will go back to overview
+        b.go("/users#/group/saturn")
+        b.wait_val("#group-name", "saturn")
+        b.click("#group-details .pf-c-dropdown button")
+        b.click("#group-details .pf-c-dropdown__menu li:contains('Delete')")
+        b.click("#group-confirm-delete-dialog footer .pf-c-button.apply")
+        b.wait_visible("#groups-list")
+        b.wait_not_in_text('#groups-list', "saturn")
+
+    def testGroupEdit(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("groupadd titan; useradd uranus")
+
+        self.addCleanup(m.execute, "groupdel titan || true")
+        self.addCleanup(m.execute, "userdel uranus || true")
+
+        self.login_and_go("/users")
+        b.click("button:contains('more...')")
+        b.wait_visible('#groups-list td[data-label="Group name"]:contains("titan")')
+        b.click("#groups-list tbody tr:contains(titan) .pf-c-dropdown button")
+        b.click("#groups-list tbody tr:contains(titan) .pf-c-dropdown__menu li:contains('Edit group')")
+
+        b.wait_val("#group-name", "titan")
+
+        # Change group name
+        b.click("#group-details .pf-c-dropdown__toggle")
+        b.click("li:contains(Rename)")
+        b.set_input_text("#group-confirm-rename-dialog #group-name", "phoebe")
```

Comment:
As a hobby physicist I really have to object here. Phoebe is a moon of Saturn (like Titan), you really can't put uranus into that group. Also, I suppose the planet should be the group and the moons the members? :moon: 

(Just kidding, of course)